dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.batchSecurity-2.0.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.batchSecurity-2.0
+visibility=private
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.batch-2.0))"
+-features=com.ibm.websphere.appserver.servlet-5.0, \
```

Comment:
batch-2.0 brings in servlet-5.0 already.  Shouldn't need to bring it in as well.

Reply:
Similar to the comment above, the previous auto feature (`...batchSecurity-1.0`) includes `servlet-3.1(4.0)`, where `batch-1.0` already includes `servlet-3.1(4.0)`.  Begs the question whether to change the previous auto feature now.

Reply:
I didn't want to mess with the tolerates one in batch 1.0 because of the non transitive nature of tolerates.